10_cdisc_questions.md

```diff
@@ -0,0 +1,68 @@
+    *** Answer: C. protection of subjects*** 
+
+ 2. Specify the variable role which focus on the observation (such as the name of a lab test) of a SDTM data set? 
+    A. Identifier
+    B. Topic
+    C. Timing
+    D. Qualifier
+    Answer: B. Topic
+
+ 3. Concomitant/Prior Medications (CM) is a component of which SDTM class?
+    A. Findings
+    B. Interventions
+    C. Events
+    D. Special Purpose
+    Answer: B. Interventions
+
+ 4. What is the primary purpose of programming validation? [comment]: <> (This is not appropriate for CDISC section, perhaps could be moved general programming section)
+    A. Ensure that the output from both the original program and the validation program match.
+    B. Efficiently ensure any logic errors are discovered early in the programming process.
+    C. Justify the means used to accomplish the outcome of a program and ensure its accurate representation of the original data.
+    D. Document all specifications pertaining to programmed output and ensure all were reviewed during the programming process.
+    Answer: C. Justify the means used to accomplish the outcome of a program and ensure its accurate representation of the original data.
+
+ 5. Why is the Demographics (DM) domain required for all human clinical trials regulatory submissions?
+    A. It is a Findings domain that contains subject-related data not collected in other domains
+    B. It is the parent domain for all other observations for human clinical subjects.
+    C. It is a Findings domain that contains general subject characteristics about demography.
+    D. It contains the actual order of elements followed by the subject.
+    Answer: B. It is the parent domain for all other observations for human clinical subjects.
+
+ 6. Which statement is correct regarding Intention-to-Treat Population? [comment]: <> (This is not appropriate for CDISC section, perhaps could be moved general programming section)
+    A. Primary analysis should include all randomised subjects.
+    B. Primary analysis should include all randomised subjects and a subject should take a single study drug.
+    C. Primary analysis should include enrolled subjects.
+    D. Primary analysis should include subjects who have taken at least a single study drug.
+    Answer: A. Primary analysis should include all randomised subjects.
+ 7. Exposure data pertaining to body temperature associated with a fever should reside in which SDTMIG domain?
+    A. Exposure (EX)
+    B. Clinical Events (CE)
+    C. Adverse Events (AE)
+    D. Vital Signs (VS)
+    Answer: D. Vital Signs (VS)
+ 8. Topic variable for the Interventions and Events General Observation Class model is often stored as _______
+    A. Standardized Value (AEDECOD, MHDECOD)
+    B. Modified Value (CMMODIFY, PEMODIFY)
+    C. Verbatim Value (AETERM, CMTRT)
+    D. Original Value (AEORRES, MHORRES)
+ Answer: C. Verbatim Value (AETERM, CMTRT)
+ 9. What is the maximum character limited allowed for ARMCD variable?

```

Comment:
28Sep: Replace with a more general question like: Classes (what is Interventional, Events, Findings, Special Domain) etc.

Reply:
Can we add some of these to the MD file? I accepted the merge